gematria/experiments/access_pattern_bm/BUILD

```diff
@@ -59,60 +70,45 @@ cc_library(
     srcs = ["vec_of_vec_matrix.cc"],
     hdrs = ["vec_of_vec_matrix.h"],
     copts = FEATURE_OPTS,
+    deps = ["@com_google_absl//absl/base:core_headers"],
 )
 
 cc_test(
-    name = "vec_of_vec_matrix_bm",
+    name = "vec_of_vec_matrix_test",
     size = "small",
-    srcs = ["vec_of_vec_matrix_test.cc"],
-    copts = select({
-        ":balance_flushing_time": ["-DBALANCE_FLUSHING_TIME"],
-        "//conditions:default": [],
-    }),
+    srcs = COMMON_TEST_HDRS + [
+        "vec_of_vec_matrix.h",
+        "vec_of_vec_matrix_test.cc",
+    ],
+    copts = BALANCE_FLUSHING_TIME_OPTS,
     deps = [
         ":vec_of_vec_matrix",
         "@com_github_google_benchmark//:benchmark_main",
     ],
 )
 
-cc_library(
-    name = "stl_container",
-    srcs = ["stl_container.cc"],
-    hdrs = ["stl_container.h"],
-    copts = FEATURE_OPTS,
-)
-
 cc_test(
-    name = "stl_container_bm",
+    name = "stl_container_test",
     size = "small",
-    srcs = ["stl_container_test.cc"],
-    copts = select({
-        ":balance_flushing_time": ["-DBALANCE_FLUSHING_TIME"],
-        "//conditions:default": [],
-    }) + FEATURE_OPTS,  # Since templated functions using _mm_clflushopt are defined in the header
+    srcs = COMMON_TEST_HDRS + [
+        "stl_container.h",
+        "stl_container_test.cc",
+    ],
+    copts = BALANCE_FLUSHING_TIME_OPTS + FEATURE_OPTS,  # Since templated functions using _mm_clflushopt are defined in the header
```

Comment:
Very nitpicky: Please add a period at the end of the comment (all comments that look like a sentence should use punctuation).

Same below.